internal/provider/helpers.go

```diff
@@ -0,0 +1,84 @@
+package provider
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/gookit/color"
+)
+
+type tfOperation string
+
+const (
+	CreateOperation tfOperation = "CREATE"
+	ReadOperation   tfOperation = "READ"
+	UpdateOperation tfOperation = "UPDATE"
+	DeleteOperation tfOperation = "DELETE"
+)
+
+func formatSQLPreview(operation tfOperation, resourceName string, id string, commands []string) string {
+	var c color.Color
```

Comment:
Instead of doing a switch every time we could define a map to just get the color from it. This is just nit-picking, it can stay like this.